src/Modules/OrchardCore.Commerce/Assets/Scripts/discount.js

```diff
@@ -0,0 +1,14 @@
+const grossPrice = document.querySelector('span.tax-part-gross-price-value');
+const defaultPrice = document.querySelector('span.price-part-price-field-value');
```

Comment:
These two elements are created dynamically (`PriceField.cshtml`), so I can't really put a shared class on them unless I check and match against the two `DisplayName()` (`GrossPrice` and `Price`) in `PriceField.cshtml`.

The other thing is, that if the product has a Tax part, then both the normal price, and the gross price is displayed, and with that approach, it would cross out both of the prices. I think it's enough to cross out the gross price if there is a tax part since the customer will only pay (in this case would have paid) for that.

Reply:
Ok, then do Array.from(document.querySelectorAll('.tax-part-gross-price-value, .price-part-price-field-value') and iterate over it.